components/ui/LeadRow/LeadRow.tsx

```diff
@@ -2,9 +2,24 @@ import React from 'react'
 import Image from 'next/image';
 import phoneImage from '/assets/images/leadslist-icons/call.png';
 import clockImage from '/assets/images/leadslist-icons/clock.png';
-import moreImage from '/assets/images/leadslist-icons/more_vert.png';
+import LeadDetailsButton from '../LeadDetailsButton/LeadDetailsButton';
 import { getStatusColor } from '@/utils/helpers/common-helpers';
 
+// const getStatusColor = (status: string): string => {
```

Comment:
If these code are not necessary then remove these.

Reply:
Done